### PR TITLE
initialize review service

### PR DESCRIPTION
**Pull request by chinathaip**

### CODE REVIEW

File: apps/backend/reviews/handlers/review.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+func (h *H) HandleGetReviews(c *fiber.Ctx) error {
+	courseCode := c.Params("courseCode")
+	ctx, cancelFunc := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancelFunc()
+
+	reviews, err := data.GetAllReviews(ctx, h.client, courseCode)
+	if len(reviews) == 0 {
+		return fiber.NewError(fiber.StatusNotFound, fmt.Sprintf("No reviews found for course %s", courseCode))
```
Comment (Lxkas) [inline apps/backend/reviews/handlers/review.go:31]: 204 better maybe? Or a 200 with no content.

File: apps/backend/reviews/routers/public_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	app    *fiber.App
+	testClient *db.PrismaClient
+)
+
+func setupTest() {
+	config.LoadEnvFile("test")
```
Comment (Lxkas) [inline apps/backend/reviews/routers/public_integration_test.go:22]: No error handling incase this fails?

Comment (chinathaip) [reply apps/backend/reviews/routers/public_integration_test.go:22]: forgot this returns error XD

File: apps/backend/reviews/handlers/review.go (hunk 1 above)
Comment (chinathaip) [reply apps/backend/reviews/handlers/review.go:31]: oki.

**Review by Lxkas — APPROVED** [review]

**Pull request merged**